src/style.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  --bigIconSize: 19px;
+  --smallIconSize: 11px;
+}
+
+.search {
+  display: flex;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.search--big {
+  height: 70px;
+  margin-top: 20px;
+}
+
+.search--small {
+  height: 42px;
+  margin-top: 20px;
+}
+
+.search input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
+}
+
+.search--big::before {
+  content: "";
+  background-image: url(images/Search.svg);
+  position: absolute;
+  z-index: 999;
+  background-size: 19px;
+  top: 25px;
+  left: 26px;
+  height: 19px;
+  width: 19px;
+
+}
+
+.search--small::before {
+  content: "";
+  background-image: url(images/Search.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  width: 11px;
+  height: 11px;
+  position: absolute;
+  top: 15px;
+  left: 13px;
+}
+
+.search input::placeholder {
+  color: #3d4e61;
+}
+
+.search--big input {
+  padding-left: 62px;
+  color: #3d4e61;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+}
+
+.search--small input {
+  padding-left: 33px;
+  color: #3d4e61;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: normal;
+}
+
+.search input:hover {
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
```

Comment:
If you want to describe one property, it is better not to use a shorthand property.
Because if there were previously declared other properties for the background, this rule clears them.